Readme.md

```diff
@@ -17,7 +17,7 @@ Collection of general purpose tools for solving problems. Fundamentally extend t
 
 ### Rust tools
 
-<!--{ generate.healthtable( 'module/core' ) } -->
+<!--{ generate.healthtable( path: module/core ) } -->
```

Comment:
should work in old way as well as in new way
also it new way should look like
<!--{ generate.healthtable( path : 'module/core' ) } -->